src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -137,6 +143,64 @@ public static String extractVersionNumber(String inputString) {
         return inputString.substring(startPos, endPos).replace("-", ".").trim();
     }
     
-    
+    public void verifyReadyForGodMode() {
+        if (!inst().isDevServer()) {
+            Assumption.fail("God mode regeneration works only in dev server.");
+        }
+        if (areTestAccountsDefaultValues()) {
+            Assumption.fail("Please change ALL the default accounts in test.properties in order to use God mode,"
+                            + "e.g change test.student1.account from alice.tmms to alice.tmms.example");
```

Comment:
The other requirement is that different developers have different test accounts so that god mode failures (if any) can also be detected by the reviewer.